### ISSUE TITLE
Provide argument for version information

### ISSUE THREAD

**Issue by hsiaoyi0504** [EddyRivasLab/hmmer#144]
This is a feature request.
I found that although there are some version information listed in help page of each compiled program.
However, there is no argument like `--version` available.
It would be good to have a argument like that to make user easier to access the version information.
It's still not very hard to parse the information to parse out version information, though.

**Comment by cryptogenomicon** [comment]
`programname -h` shows both help and version information.

**Comment by hsiaoyi0504** [comment]
Yep, as I mentioned that there are "there are some version information listed in help page of each compiled program", but my point is that if someone want to implement feature like checking the program version used, he would need to further parse the help message. It would be a little inconvenient. It's about building the application on top of hmmer, not about if user can get the pretty the same information.

**Comment by cryptogenomicon** [comment]
You need to do so with any program, because formatting of this info, and what option produces it, varies. Here is `gcc --version` on OS/X, for example.

```
Configured with: --prefix=/Applications/Xcode-beta.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 9.1.0 (clang-902.0.30)
Target: x86_64-apple-darwin17.6.0
Thread model: posix
InstalledDir: /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

**Comment by hsiaoyi0504** [comment]
Ok, thanks. I got your point.

**Comment by cryptogenomicon** [comment]
(And it's not just _some_ version info, it's _complete_ version info that's in `progname -h` output.)

**Comment by hsiaoyi0504** [comment]
Thanks for pointing out that, it's my fault to use the word inaccurately.
